lib/uploadcare/client/conversion/base_conversion_client.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative '../rest_client'
+require 'exception/conversion_error'
+
+module Uploadcare
+  module Client
+    module Conversion
+      # This is a base client for conversion operations
+      #
+      # @see https://uploadcare.com/api-refs/rest-api/v0.6.0/#tag/Conversion
+      class BaseConversionClient < RestClient
+        def headers
+          {
+            'Content-type': 'application/json',
```

Comment:
```suggestion
            'Content-Type': 'application/json',
```

Reply:
The _titleized_ `Type` is not accepted by the API correctly. Invalid signature error occurs. @dmitry-mukhin, what do you think about this header?

Reply:
signature should be made from exact headers that are being sent. signature validation does not care about headers' capitalisation.
Can you demonstrate the issue?

Reply:
here's the source of the bug:
https://github.com/uploadcare/uploadcare-ruby/blob/feature/5-document-conversion-convert/lib/uploadcare/client/rest_client.rb#L27-L28
```
        request_headers = Param::AuthenticationHeader.call(method: method.upcase, uri: uri,
                                                           content_type: headers[:'Content-type'], **options)
```

Reply:
so yeah, fix capitalisation in both places :]

Reply:
@dmitry-mukhin, problem is here https://github.com/uploadcare/uploadcare-ruby/blob/feature/5-document-conversion-convert/lib/uploadcare/client/rest_client.rb#L31

when I set Content-Type in the `headers` method (https://github.com/uploadcare/uploadcare-ruby/blob/feature/5-document-conversion-convert/lib/uploadcare/client/rest_client.rb#L57) and try to convert a video I get
```console
Invalid signature. Please check your secret key.
```
And when `Content-type` header is set - everything works fine

Reply:
This may be out of scope of this PR, but header capitalization is internal to the lib, nothing stops us from fixing this.

This is why in good http related libs all header names are dynamically cast to lowercase or uppercase so strings with mixed casing from user or external entities do not cause issues.